Sources/StreamChat/Query/ChannelMemberListQuery.swift

```diff
@@ -13,38 +13,69 @@ public class MemberListFilterScope: UserListFilterScope, AnyMemberListFilterScop
 
 /// Non extra-data-specific filer keys for member list.
 public extension FilterKey where Scope: AnyMemberListFilterScope {
+    #warning("Tricky! Test filter generation for BE as well as predicate generation")
     /// A filter key for matching moderators of a channel.
     /// Supported operators: `equal`, `notEqual`
-    static var isModerator: FilterKey<Scope, Bool> { "is_moderator" }
+    static var isModerator: FilterKey<Scope, Bool> {
+        .init(
+            payloadKey: MemberPayload.CodingKeys.role,
+            dtoKey: #keyPath(MemberDTO.channelRoleRaw)
+//            value: MemberRole.moderator.rawValue
```

Comment:
nit: Can we remove this if it's not needed?

Reply:
This should not be commented out, actually 🤔 Any reason why it is now? @polqf